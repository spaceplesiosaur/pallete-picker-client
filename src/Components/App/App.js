import React from 'react';
import ProjectsContainer from '../ProjectsContainer/ProjectsContainer';

import './App.scss';

import Nav from '../Nav/Nav';
import PalettesContainer from '../PalettesContainer/PalettesContainer';

function App() {
  return (
    <main className="App">
       <Nav />
       <section className='palette-info'>
          <PalettesContainer />
        </section>
       <section className='main-project-info'>
         <ProjectsContainer />
       </section>
    </main>
  );
}

export default App;
