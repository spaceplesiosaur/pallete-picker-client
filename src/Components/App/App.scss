* {
  margin: 0;
  padding: 0;
  font-family: 'Roboto Condensed', sans-serif;
  box-sizing: border-box;
}

.App {
  display: flex;
  flex-direction: column;
  text-align: center;
  color: #FCC606;
}

.palette-info {
  display: flex;
  background-image: url('https://ak9.picdn.net/shutterstock/videos/28651609/thumb/1.jpg');
    color: #FCC606;
    background-repeat: no-repeat;
    background-size: cover;
    background-position: center;
    height: 100vh;
}

.main-project-palette {
  display: flex;
  flex-direction: row;
  text-align: center;
}
