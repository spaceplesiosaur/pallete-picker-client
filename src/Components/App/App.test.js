import React from 'react';
// import { render } from '@testing-library/react';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App
      />)
  })

  it('should render with the correct properites', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
