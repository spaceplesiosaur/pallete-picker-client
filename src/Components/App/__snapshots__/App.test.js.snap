// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render with the correct properites 1`] = `
<main
  className="App"
>
  <Nav />
  <section
    className="palette-info"
  >
    <PalettesContainer />
    <Connect(ProjectsContainer) />
  </section>
</main>
`;
